components/micro-gateway-cli/src/main/resources/templates/prodEndpointReinit.mustache

```diff
@@ -0,0 +1,39 @@
+etcdKeyConfigValue = apiId + "_prod_etcdKey";
```

Comment:
Can we think of a way to avoid duplicating code segments in this file as well as in the sandboxendpoint file? What if we have another common file where we pass the prod or sandbox endpoint to the common file.